### ISSUE TITLE
Remove the NFV profile and its example

### ISSUE THREAD

**Issue by philippemerle** [oasis-open/tosca-community-contributions#111]
We should remove the NFV profile and its example:
* https://github.com/oasis-open/tosca-community-contributions/tree/master/profiles/org.oasis-open/simple-for-nfv/1.0
* https://github.com/oasis-open/tosca-community-contributions/tree/master/examples/1.2/simple-for-nfv

OASIS never released the NFV profile as a standard (we stopped work after CSC04) and the work was taken over by ETSI.

**Comment by lauwers** [comment]
I agree. Unless there are any objections, we will do this over the next couple of days.

**Comment by lauwers** [comment]
This example has been removed.
